Ginger/GingerCoreNET/Application Models/Execution/POM/POMExecutionUtils.cs

```diff
@@ -84,6 +84,20 @@ public ElementInfo GetCurrentPOMElementInfo()
             return selectedPOMElementInfo;
         }
 
+        public ElementInfo GetFriendlyElementInfo(Guid elementGuid)
+        {
+            Guid currentPOMElementInfoGUID = elementGuid;
+            ElementInfo selectedPOMElementInfo = GetCurrentPOM().MappedUIElements.Where(z => z.Guid == currentPOMElementInfoGUID).FirstOrDefault();
+
+            if (selectedPOMElementInfo == null)
+            {
+                mAct.ExInfo = string.Format("Failed to find the mapped element with GUID '{0}' inside the Page Objects Model", selectedPOMElementInfo.ToString());
```

Comment:
selectedPOMElementInfo.tostring will throw error because it is null

Reply:
Done